generator/src/test/java/com/scottlogic/deg/generator/cucumber/Before.feature

```diff
@@ -19,124 +19,7 @@ Scenario: Running a 'before' request that specifies a valid date should be succe
        | 2019-02-09T23:59:59.997 |
        | 2019-02-09T23:59:59.996 |
 
-# Defect 600 "Running a before constraint in full generation mode generates interesting data" related to this scenario
-@ignore
-Scenario: Running a 'before' request that specifies a valid time should be successful
```

Comment:
Should all of these scenarios be removed?

Reply:
Covered scenario elsewhere in this suite